src/OCServo.h

```diff
@@ -0,0 +1,89 @@
+#ifndef OCSERVO_H
+#define OCSERVO_H
+
+#include <Arduino.h>
+#include <SoftwareSerial.h>
+
+#include "ROBSMotor.h"
+
+enum OCSMode {
+    OCS_SERVO,
+    OCS_MOTOR
+};
+
+enum OCSResponseLevel {
+    OCS_RESPOND_READ_ONLY,
+    OCS_RESPOND_ALL
+};
+
+typedef enum e_SerialPortType {
+    SERIAL_HARDWARE,
+    SERIAL_SOFTWARE
+} SerialPortType;
```

Comment:
The SerialPortType typedef could be unnecessary. Using enum SerialPortType directly is more standard and does not require the typedef keyword in modern C++.
```suggestion
/**
 * @brief Type of serial port used by OCServo
 */
enum class SerialPortType {
    Hardware,
    Software
};
```